README.md

```diff
@@ -1,4 +1,7 @@
-# Consul + Kubernetes (consul-k8s)
+<h1>
+  <img src="./assets/logo.svg" align="left" height="46px" alt="Consul logo"/>
+  <span>Consul + Kubernetes</span>
```

Comment:
```suggestion
  <span>Consul on Kubernetes</span>
```

Reply:
Agreed, I also have been thinking about this, but I didn't want to rock the boat. I only rock the vote.